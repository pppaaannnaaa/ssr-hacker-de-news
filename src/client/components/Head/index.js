import React from 'react';
import { Helmet } from 'react-helmet';

const head = () => {
  return (
    <Helmet key={Math.random()}>
      <title>News de Hacker</title>
      <meta property="og:title" content="News de Hacker" />
      <meta
        name="description"
        content="News de Hacker Server side rendering application with React JS"
      />
      <meta name="robots" content="index, follow" />
      <link rel="canonical" href="#" />
    </Helmet>
  );
};

export default head;
