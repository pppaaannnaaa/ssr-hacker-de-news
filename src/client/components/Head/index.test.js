import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Head from './index';

describe('<Head />', () => {
  let props;

  beforeEach(() => {
    props = {};
  });

  afterEach(cleanup);

  test('Renders <Head /> component correctly', () => {
    const { container } = render(<Head />);
    expect(container).toMatchSnapshot();
  });
});
