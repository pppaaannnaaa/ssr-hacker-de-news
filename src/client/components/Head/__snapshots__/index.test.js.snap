// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Head /> Renders <Head /> component correctly 1`] = `<div />`;
